common/os.go

```diff
@@ -17,7 +18,7 @@ var (
 func TrapSignal(cb func()) {
 	c := make(chan os.Signal, 1)
 	signal.Notify(c, os.Interrupt)
-	signal.Notify(c, os.Kill)
+	signal.Notify(c, syscall.SIGTERM)
```

Comment:
this is recomended by the linter? any idea why? os.Kill is just an alias for syscall. SIGTERM, ie. https://golang.org/pkg/os/#Signal

Reply:
the error is "cannot trap ..." something something something. I made this fix in tendermint core awhile back and in various places in ipfs

Reply:
https://github.com/ipfs/go-ipfs/pull/3949/files#r119514774

Reply:
huh - I think this is actually a bug in go-vet than. os.Kill can be trapped - it's literally just an alias for syscall.SIGTERM. Maybe you want to alert them :)

Reply:
reverted